draft-ietf-capport-architecture.xml

```diff
@@ -883,30 +869,27 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
             </abstract>
           </front>
         </reference>
-        <reference anchor="RFC7710" target="https://www.rfc-editor.org/info/rfc7710" xml:base="https://xml2rfc.tools.ietf.org/public/rfc/bibxml/reference.RFC.7710.xml">
-          <front>
-            <title>Captive-Portal Identification Using DHCP or Router Advertisements (RAs)</title>
-            <seriesInfo name="DOI" value="10.17487/RFC7710"/>
-            <seriesInfo name="RFC" value="7710"/>
-            <author initials="W." surname="Kumari" fullname="W. Kumari">
-              <organization/>
-            </author>
-            <author initials="O." surname="Gudmundsson" fullname="O. Gudmundsson">
-              <organization/>
-            </author>
-            <author initials="P." surname="Ebersman" fullname="P. Ebersman">
-              <organization/>
-            </author>
-            <author initials="S." surname="Sheng" fullname="S. Sheng">
-              <organization/>
-            </author>
-            <date year="2015" month="December"/>
-            <abstract>
-              <t>In many environments offering short-term or temporary Internet access (such as coffee shops), it is common to start new connections in a captive-portal mode.  This highly restricts what the customer can do until the customer has authenticated.</t>
-              <t>This document describes a DHCP option (and a Router Advertisement (RA) extension) to inform clients that they are behind some sort of captive-portal device and that they will need to authenticate to get Internet access.  It is not a full solution to address all of the issues that clients may have with captive portals; it is designed to be used in larger solutions.  The method of authenticating to and interacting with the captive portal is out of scope for this document.</t>
-            </abstract>
-          </front>
-        </reference>
+	<reference anchor='RFC7710bis'>
```

Comment:
I had been looking for a way to make the <? rfc ...refrence.I-D.ietf-capport-7710bis.xml ?> style of reference work, but my xml2rfc fu was too weak.